develop/tutorials/articles/115-mobile/01-android-apps-with-liferay-screens/07-creating-android-screenlets.markdown

```diff
@@ -177,29 +179,29 @@ to handle user actions. Follow these steps to create this View class:
     has a reference to the Screenlet. For example, here are `AddBookmarkView`'s 
     instance variables: 
 
-        private EditText _urlText;
-        private EditText _titleText;
-        private BaseScreenlet _screenlet;
+        private EditText urlText;
```

Comment:
Whoa; the Screens team is departing from the Liferay code style of adding an underscore for instance variables?